buckets-mdapi/src/object.rs

```diff
@@ -1,14 +1,15 @@
 // Copyright 2020 Joyent, Inc.
+// Copyright 2023 MNX Cloud, Inc.
 
 use std::error::Error;
 use std::vec::Vec;
 
 use base64;
 use bytes::BytesMut;
+use postgres::types::{accepts, to_sql_checked};
```

Comment:
Why did this move?

Reply:
`cargo fmt` will sort `use` statements alphabetically